src/child/containers/sidebars/favourites/Favourites.js

```diff
@@ -88,6 +88,7 @@ class Favourites extends Component {
                               stockCode={stockCode}
                               bindings={bindings}
                               selected={stockCode === selection.code}
+                              favourites={favourites}
```

Comment:
Exposing the Favourites to the Favourite seems excessive, can we move the code to the toggleFavourite fn (and possibly rename it)?